lib/temperature/index.js

```diff
@@ -18,7 +18,13 @@
 'use strict';
 
 var console = require('console');
-var BMP085 = require('bmp085-sensor');
+try {
+  var BMP085 = require('bmp085-sensor');
+} catch (err){
+  // console.log(err);
+}
+
+var SIMULATOR = require('./simulator.js');
```

Comment:
Generally, in JS just the 1st letter is used for Classes, expect BMP which is an acronym